util/execute-deploy.sh

```diff
@@ -50,6 +51,12 @@ deploy_library \
   dagger.pom.xml
 
 deploy_library \
+  java/dagger/libgwt.jar \
+  java/dagger/libgwt.jar \
```

Comment:
Do these not need sources jars or javadoc jars? I guess they don't really have any...